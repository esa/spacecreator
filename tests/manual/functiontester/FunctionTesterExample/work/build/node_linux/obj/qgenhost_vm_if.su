qgenhost_vm_if.c:13:6:init_qgenhost	32	static
qgenhost_vm_if.c:25:6:qgenhost_Calculate	80	static
qgenhost_vm_if.c:56:1:_GLOBAL__sub_D_00099_0_qgenhost_initialized	16	static
qgenhost_vm_if.c:56:1:_GLOBAL__sub_I_00099_1_qgenhost_initialized	16	static
