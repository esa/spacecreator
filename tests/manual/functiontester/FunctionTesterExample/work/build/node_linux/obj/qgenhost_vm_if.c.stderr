[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:[m[K In function â€˜[01m[Kinit_qgenhost[m[Kâ€™:
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:20:7:[m[K [01;35m[Kwarning: [m[Kimplicit declaration of function â€˜[01m[Kqgenhost_PI_Calculate_startup[m[Kâ€™; did you mean â€˜[01m[Kqgenhost_Calculate[m[Kâ€™? [[01;35m[K-Wimplicit-function-declaration[m[K]
       [01;35m[Kqgenhost_PI_Calculate_startup[m[K();
       [01;35m[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~[m[K
       [32m[Kqgenhost_Calculate[m[K
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:17:17:[m[K [01;35m[Kwarning: [m[Kunused variable â€˜[01m[Kunused[m[Kâ€™ [[01;35m[K-Wunused-variable[m[K]
       long long [01;35m[Kunused[m[K = getTimeInMilliseconds();
                 [01;35m[K^~~~~~[m[K
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:[m[K In function â€˜[01m[Kqgenhost_Calculate[m[Kâ€™:
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:26:40:[m[K [01;35m[Kwarning: [m[Kunused parameter â€˜[01m[KIN_intparam_len[m[Kâ€™ [[01;35m[K-Wunused-parameter[m[K]
       (const char *IN_intparam, [01;35m[Ksize_t IN_intparam_len[m[K,
                                 [01;35m[K~~~~~~~^~~~~~~~~~~~~~~[m[K
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:27:41:[m[K [01;35m[Kwarning: [m[Kunused parameter â€˜[01m[KIN_realparam_len[m[Kâ€™ [[01;35m[K-Wunused-parameter[m[K]
        const char *IN_realparam, [01;35m[Ksize_t IN_realparam_len[m[K,
                                  [01;35m[K~~~~~~~^~~~~~~~~~~~~~~~[m[K
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:28:41:[m[K [01;35m[Kwarning: [m[Kunused parameter â€˜[01m[KIN_boolparam_len[m[Kâ€™ [[01;35m[K-Wunused-parameter[m[K]
        const char *IN_boolparam, [01;35m[Ksize_t IN_boolparam_len[m[K,
                                  [01;35m[K~~~~~~~^~~~~~~~~~~~~~~~[m[K
[01m[K/home/taste/FunctionTesterExample/work/qgenhost/QGenC/wrappers/qgenhost_vm_if.c:29:36:[m[K [01;35m[Kwarning: [m[Kunused parameter â€˜[01m[KOUT_outparam_len[m[Kâ€™ [[01;35m[K-Wunused-parameter[m[K]
        char *OUT_outparam, [01;35m[Ksize_t *OUT_outparam_len[m[K)
                            [01;35m[K~~~~~~~~^~~~~~~~~~~~~~~~[m[K
